docker/README.md

```diff
@@ -0,0 +1,50 @@
+# Docker
```

Comment:
Just to note, prettier will probably want to reformat this (blank lines between instructions would help me a lot, and prettier should add that). I don't know why pre-commit is hung on this PR, but it's been long enough that I don't know how to investigate.

Reply:
Yea no problem at all as I have been busy myself.
I can try fixing it on the next commit.

Reply:
Did you proceed with using apt instead of homebrew? Just now getting back into this topic.

Reply:
Yes, and looks like you found the devcontainer file